lib/main.coffee

```diff
@@ -31,9 +26,20 @@ module.exports =
     @subscriptions.add atom.config.observe 'linter-phpmd.rulesets',
       (rulesets) =>
         @rulesets = rulesets
-    @subscriptions.add atom.config.observe 'linter-phpmd.projectRules',
-      (projectRules) =>
-        @projectRules = projectRules
+    if atom.config.get('linter-phpmd.projectRules') == true &&
+    atom.config.get('linter-phpmd.rulesets') ==
```

Comment:
This could just be GitHub's web interface, but the indentation looks off for this whole block...